backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationMutationResolver.java

```diff
@@ -426,10 +429,20 @@ public OrganizationQueueItem markPendingOrganizationAsDeleted(
   }
 
   /** Support-only mutation to mark an organization as deleted. This is a soft deletion only. */
+  @Transactional
```

Comment:
What is the reason for taking off @Transactional?

Reply:
If I'm understanding `@Transactional` correctly, I added it here just in case the other delete operations failed I didn't want to leave the org with some part of it (the org itself, its users, or its facilities) deleted (or undeleted) successfully while other parts of it not.

Reply:
Ignore my comment, I read the changes incorrectly (apparently struggling to see the difference between red and green lol), this makes sense to me! 😸